eventdata/runners/deleteindex_runner.py

```diff
@@ -66,7 +66,8 @@ def get_suffix(name, separator):
     suffix_separator = params.get('suffix_separator', '-')
 
     if max_indices:
-        indices = await es.cat.indices(h='index').split("\n")
+        response = await es.cat.indices(h='index')
```

Comment:
should this be double quotes here as well?

Reply:
Good point. I tried to change as little as possible and the eventdata track is quite inconsistent in any case but let's change this here.

Reply:
I've pushed https://github.com/elastic/rally-eventdata-track/pull/82/commits/07332a787a9c056205cae5f5ebe2cb6a8ae89aac.